server-kotlin/src/main/kotlin/digital/cesko/internet_stream/InternetStreamService.kt

```diff
@@ -151,7 +150,8 @@ class InternetStreamService(
 
     fun saveCityBudgets(cityId: Int, allBudgets: List<Budget>) {
         allBudgets.chunked(threshold) {
-            budgets: List<Budget> -> insertIntoPayments(cityId, budgets)
+            budgets: List<Budget> ->
```

Comment:
This would be usually written after the curly brace `allBudgets.chunked(threshold) { budgets -> `